core/src/main/kotlin/org/evomaster/core/EMConfig.kt

```diff
@@ -1420,9 +1423,6 @@ class EMConfig {
     @Min(-1.0)
     var maxLengthOfTraces = 10
 
-    @Cfg("Enable custom naming and sorting criteria")
```

Comment:
instead of removing it, to avoid breaking clients that use it, rather put it to false, and mark it with a `@Deprecated("No longer in use")`

Reply:
Done